resources/mac/Info.plist

```diff
@@ -20,9 +20,12 @@
     <string>@SHORT_VERSION@</string>
     <key>CFBundleVersion</key>
     <string>@SHORT_VERSION@</string>
+    <key>CFBundleGetInfoString</key>
+    <string>ghostwriter @SHORT_VERSION@</string>
     <key>CFBundleName</key>
-    <string>ghostwriter</string>
-
+    <string>Ghostwriter</string>
```

Comment:
App name should be in lowercase:

```
<string>ghostwriter</string>
```

Reply:
This is the App name used in mac system toolbar. Is a convention to use Uppercase names here. The executable will be lowercase.

Reply:
Interesting.  Okay.  Out of idle curiosity, does iA Writer show up as "IA Writer" in the mac system toolbar?  Just wondering if the rules can be bent.  I understand if they can't be.

Reply:
No idea, but iTunes shows up as "iTunes", so the same can be applied to ghostwriter :p